src/Model.jl

```diff
@@ -240,3 +246,67 @@ function compute_unit_cell_volume(lattice)
     n_dim = count(!iszero, eachcol(lattice))
     abs(det(lattice[1:n_dim, 1:n_dim]))
 end
+
+
+function Base.show(io::IO, model::Model)
+    nD = model.n_dim == 3 ? "" : "$(model.n_dim)D, "
+    print(io, "Model(", model.name, ", ", nD,
+          "spin_polarization = ", model.spin_polarization, ")")
+end
+
+
+function Base.show(io::IO, ::MIME"text/plain", model::Model)
+    println(io, "Model(", model.name, ", $(model.n_dim)D):")
+    for i = 1:3
+        header = i==1 ? "lattice (in Bohr)" : ""
+        showfieldln(io, header, (@sprintf "[%-10.6g, %-10.6g, %-10.6g]" model.lattice[i, :]...))
+    end
+    showfieldln(io, "unit cell volume", @sprintf "%.5g Bohr³" model.unit_cell_volume)
+
+    if !isempty(model.atoms)
+        println(io)
+        showfieldln(io, "atoms", sum_formula(model.atoms))
+        elements = first.(model.atoms)
+        for (i, el) in enumerate(elements)
+            header = i==1 ? "atom potentials" : ""
+            showfieldln(io, header, el)
+        end
+    end
+
+    println(io)
+    showfieldln(io, "num. electrons",    model.n_electrons)
+    showfieldln(io, "spin polarization", model.spin_polarization)
+    showfieldln(io, "temperature",       @sprintf "%.5g Ha" model.temperature)
+    if model.temperature > 0
+        showfieldln(io, "smearing",      model.smearing)
+    end
+
+    println(io)
+    for (i, term) in enumerate(model.term_types)
+        header = i==1 ? "terms" : ""
+        showfield(io, header, term)
+        i < length(model.term_types) && println(io)
+    end
+end
+
+
+
+"""
+Returns the sum formula of the atoms list as a string.
+"""
+function sum_formula(atoms)
```

Comment:
is "sum formula" a standard term? It doesn't seem to be. Chemical formula? (also add a dispatch on sum_formula(::Model))

Reply:
It is in German :P But the English term is "molecular formula", so I'll use that.